sources/platform/actors/development/programming_interface/basic_commands.md

```diff
@@ -12,17 +12,18 @@ import TabItem from '@theme/TabItem';
 
 ---
 
-The commands described in this section are expected to be called from within a context of a running Actor, both in a local environment or on the Apify platform.
+This page covers essential commands for the Apify SDK in JavaScript & Python. These commands are designed to be used within a running Actor, either in a local environment or on the Apify platform.
 
-## Actor initialization
+## Initialize your Actor
 
-First, the Actor should be initialized. During initialization, it prepares to receive events from the Apify platform. The Actor determines the machine and storage configuration and optionally purges the previous state from local storage. It will also create a default instance of the Actor class.
+Before using any Apify SDK methods, initialize your Actor. This step prepares the Actor to receive events from the Apify platform, sets up machine and storage configurations, and optionally clears previous local storage states.
 
-It is not required to perform the initialization explicitly because the Actor will initialize on the execution of any Actor method, but we strongly recommend it to prevent race conditions.
+<Tabs groupId="main">
+<TabItem value="JavaScript" label="JavaScript">
 
-### Using the JavaScript SDK
+Use the `init()` method to initialize your Actor. Pair it with `exit()` to properly terminate the Actor
 
-The Actor is initialized by calling the `init()` method. It should be paired with an `exit()` method, which terminates the Actor. The use of `exit()` is not required but recommended. For more information, go to [Exit Actor](#exit-actor).
+The use of `exit()` is not required but recommended. For more information, go to [Exit Actor](#exit-actor).
```

Comment:
the exit call is kinda required, if you use `init`, you open websocket connection, and without the `exit` call your program would not finish. at least that's how it used to work and i am not aware of any changes.